packages/e2e-tests/specs/blocks/table.test.js

```diff
@@ -10,6 +15,17 @@ import {
 
 const createButtonSelector = "//div[@data-type='core/table']//button[text()='Create Table']";
 
+/**
+ * Utility function for changing the selected cell alignment.
+ *
+ * @param {string} align The alignment (one of 'left', 'center', or 'right').
+ */
+async function changeCellAlignment( align ) {
+	await clickBlockToolbarButton( 'Change column alignment' );
+	const alignButton = await page.$x( `//button[text()='Align Column ${ capitalize( align ) }']` );
```

Comment:
Can the `clickButton` utility function be used here?

Reply:
Genuinely didn't know that existed. Will do a follow up PR to tidy up the tests.